pom.xml

```diff
@@ -1,144 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why was this file removed? Not needed?

Reply:
Yes, currently we are creating pom.xml on the fly to publish to maven repo